OleksandraKotovska/HW3/1.py

```diff
@@ -0,0 +1,22 @@
+main_str = """
+Beautiful is better than ugly.
+Explicit is better than implicit.
+Simple is better than complex.
+Complex is better than complicated.
+Flat is better than nested.
+Sparse is better than dense.
+Readability counts.
+Special cases aren't special enough to break the rules.
+Although practicality beats purity.
+Errors should never pass silently.
+"""
+print(main_str.count('better'))
```

Comment:
good job

look at this version of code

words_to_count = ['better', 'never', 'is']

word_counts = {word: main_str.count(word) for word in words_to_count}

for word, count in word_counts.items():
    print(f'{word} - {count}')